HED-schema-reduced.mediawiki

```diff
@@ -158,47 +166,32 @@ There are four ways to define the scope of an event.  The most common method is
 *** Clear throat
 *** Yawn 
 *** Burp 
-*** Flex
-*** Jerk <nowiki>[When some force applied to the participant causes them to involuntary accommodate or react to it.]</nowiki> 
-*** Stumble <nowiki>[Temporary and involuntary loss of balance]</nowiki>
-***	Recover balance <nowiki>[A movement made in response to a stumble or unforeseen loss of balance]</nowiki>
-*** Fall <nowiki>[Involuntary fall to floor or ground]</nowiki>
-*** Stretch <nowiki>[Stretching the body or body part]</nowiki> 
-*** Bend 
-*** Scratch 
-*** Lift
-*** Tic
-* Communicative
-** Verbal
+
+
+* Communicate
+** Communicate verbally
 *** Speak <nowiki>[Communicate using spoken language.]</nowiki>
-** Interjective
 *** Groan
 ***	Laugh
 *** Scream
 *** Sigh
-** Gestural <nowiki>[Make a expressive communicative movement of hands/arms and, often, face]</nowiki>
+** Gesture <nowiki>[Make a expressive communicative movement of hands/arms and, often, face]</nowiki>
 ***	Shrug
 ***	Thumb(s) up
-***	Fist pump
-***	Hand spread
+***	Pump fist
+***	Spread hand
 ***	Wave
 ***	Index up
 ***	Shhh
-** Musical
+** Communicate musically
 *** Hum 
 *** Whistle
 *** Vocalize <nowiki>[Make a sustained vowel sound such as, Ahhh]</nowiki>
 *** Sing
 *** Perform
 
-* Mental
+* Think
```

Comment:
My two cents is that "Think" is not quite general enough. How about "Cerebrate"?...

Reply:
It's a good word, but I had to look it up. It looks too much like celebrate.  Alternatives are Deliberate, Cogitate, Contemplate, Ponder, Mull over  (from the thesaurus).

Reply:
Maybe Deliberate... All the other words seem too specific (hinting at intense and long thinking)